### ISSUE TITLE
Gles2 3d lighting corrupted

### ISSUE THREAD

**Issue by Luigi654** [godotengine/godot#50189]
### Godot version

3.3.2 stable

### System information

Win7, gles2, videocard: intel GMA 3600 series

### Issue description

The lighting in gles2 in 3d makes me my model invisible or destroyed and only need use unshaded to work :(

Also causes my videocard shutdown and bluescreen

### Steps to reproduce

Load a .obj file and put light in the model

**Comment by clayjohn** [comment]
Do you get any error messages printed to the console when that happens?

**Comment by Luigi654** [comment]
> Do you get any error messages printed to the console when that happens?

No, only in the console happens as "da vinci code" at edit material

**Comment by Calinou** [comment]
Very old Intel IGPs have known issues with Godot, even when using GLES2. There is nothing we can do about it.

**Comment by Luigi654** [comment]
> Very old Intel IGPs have known issues with Godot, even when using GLES2. There is nothing we can do about it. You can install a [software OpenGL implementation](https://github.com/pal1000/mesa-dist-win) for Godot, which will make rendering correct but very slow.

Thanks n.n
